less/v2/inventory.less

```diff
@@ -51,13 +51,15 @@
       }
 
       .breakpoint {
+        --encumbrance-low: 33%;
+        --encumbrance-high: 66%;
```

Comment:
If you move this up and define these variables on `.meter` instead, then you can set them in the template at the same declaration where we set `--bar-percentage` and avoid setting them twice within the template.